### ISSUE TITLE
Replace eslint style rules with prettier

### ISSUE THREAD

**Issue by alexreardon** [atlassian/react-beautiful-dnd#552]
Right now we have a lot of [eslint rules](https://github.com/atlassian/react-beautiful-dnd/blob/master/.eslintrc) including extending the airbnb rules.

I am keen to move *all* of the style rules to prettier (eg our custom eslint rule for dangling comma's)

However, I am keen to keep the eslint rules that are non-style related. Things like eslint-plugin-jsx-a11y and so on.

Here is my proposed `.prettierrc` config

```
{
  "trailingComma": "all",
  "semi": true,
  "tabWidth": 2,
  "useTabs": false,
  "singleQuote": true
}
```

**Comment by alexreardon** [comment]
This work should be done after #493, and probably at the same time as #525

**Comment by alexreardon** [comment]
Shipped in #581
